fdbserver/Status.actor.cpp

```diff
@@ -3135,6 +3166,8 @@ ACTOR Future<StatusReply> clusterGetStatus(
 
 		state JsonBuilderObject idmpKeyStatus = wait(getIdmpKeyStatus(cx));
 
+		state JsonBuilderObject versionEpochStatus = wait(versionEpochStatusFetcher(cx, &status_incomplete_reasons));
```

Comment:
Can we wait this along with other actors, e.g., `idmpKeyStatus`? We want to parallelize them to reduce the latency for status.